shell/common/engine.cc

```diff
@@ -197,23 +199,27 @@ void Engine::RunFromSnapshotStream(
 void Engine::ConfigureAssetBundle(const std::string& path) {
   struct stat stat_result = {0};
 
+  directory_asset_bundle_.reset();
+  zip_asset_bundle_.reset();
+
   if (::stat(path.c_str(), &stat_result) != 0) {
     LOG(INFO) << "Could not configure asset bundle at path: " << path;
     return;
   }
 
   if (S_ISDIR(stat_result.st_mode)) {
     // Directory asset bundle.
-    new blink::DirectoryAssetBundle(mojo::GetProxy(&root_bundle_), path,
-                                    blink::Threads::IO());
+    directory_asset_bundle_ = std::make_unique<blink::DirectoryAssetBundle>(
```

Comment:
Yeah, there's a growing consensus that StrongBinding is a bad pattern, which is why I took this opportunity to remove it.